app/workers/deferred_dispatch.rb

```diff
@@ -12,12 +12,31 @@ def perform(user_id, object_class_name, object_id, opts)
       opts = HashWithIndifferentAccess.new(opts)
       opts[:services] = user.services.where(:type => opts.delete(:service_types))
 
+      if AppConfig.relay.outbound.send? && object_class_name == "StatusMessage"
+        handle_relay(object, opts)
+      end
+
       if opts[:additional_subscribers].present?
         opts[:additional_subscribers] = Person.where(:id => opts[:additional_subscribers])
       end
-
       Postzord::Dispatcher.build(user, object, opts).post
     rescue ActiveRecord::RecordNotFound # The target got deleted before the job was run
     end
+
+    def handle_relay(object, opts)
+      if object.respond_to?(:public?) && object.public?
+        relay_contact = Person.find_by diaspora_handle: AppConfig.relay.outbound.handle.to_s
+        if relay_contact
+          if opts[:additional_subscribers].present?
+            opts[:additional_subscribers] += [relay_contact.id]
+          else
+            opts[:additional_subscribers] = [relay_contact.id]
+          end
```

Comment:
I feel we should stay with `relay_person` since `contact` is a specific entity in the codebase.